sat/apiclient/fas.py

```diff
@@ -354,6 +376,8 @@ def _create_temporary_snapshot(self, xnames=None):
             except KeyError:
                 raise APIError('Payload returned from GET to snapshots/name did not have "ready" field.')
 
+        LOGGER.info(f'Use `sat firmware --delete-snapshot` command to delete the snapshot if it is no longer needed')
```

Comment:
I think you can drop "command" from this message.

```suggestion
        LOGGER.info(f'Use `sat firmware --delete-snapshot` to delete the snapshot if it is no longer needed')
```